extensions/kubernetes/openshift/deployment/pom.xml

```diff
@@ -49,6 +49,10 @@
             <artifactId>quarkus-junit5-internal</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye.config</groupId>
+            <artifactId>smallrye-config-source-yaml</artifactId>
```

Comment:
this dependency is added on purpose? Why do we need it?

Reply:
Sorry, this was meant to be test scope.